lib/utils/config.ts

```diff
@@ -84,6 +89,12 @@ export async function generateApplicationConfig(
 			: options.os.sshKeys;
 	}
 
+	// configure installer secure boot opt-in if specified
+	if (options.secureBoot) {
+		config.installer = config.installer ? config.installer : {};
```

Comment:
I think this can be simplified as 
```suggestion
		config.installer = config.installer ?? {};
```

Reply:
Or even shorter:
```suggestion
		config.installer ??= {};
```

Reply:
Done, thanks.